### ISSUE TITLE
[Web] [Accessibility] [11yMAS][Bypass Blocks] [MAS2.4.1] [Skip to main content link is not provided which helps to Skip header controls.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#687]
**User Experience:**
User will face difficulty if “Skip to main content” link is not provided which helps user to skip the repetitive content[Headers] every time and navigate directly to main content.
 
**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0
**Attachments**
 [MAS2.4.1_Skip to Main content.zip](https://github.com/OfficeDev/script-lab/files/1445245/MAS2.4.1_Skip.to.Main.content.zip)

**Repro Steps:**
1. Open the URL: http://scriptlabdocsdemo.azurewebsites.net/Office%20Dev%20Center%20-%20Docs%20-%20Build%20your%20first%20Excel%20add-in.html .
2. Navigate through the page Header section.
3. Verify ”skip to main content “ link is provided or not. 
 
**Current Behavior:** 
“Skip to main content” link is not provided which helps user to skip the repetitive content and navigate directly to main content

**Expected Behavior:**
“Skip to main content” link should be provided so that user can  skip the repetitive content(headers) every time and navigate directly to main content
**Note:**
1.Same issue is repro for the [MAC + safari][ Build info: OS Build: Mac OS Sierra 0.12.6 Browser: Safari 11.0.1] Skip to main content link is not provided which helps to skip header controls
**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={20bce2aa-9952-4a10-af79-3376bcc8766d} 
**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=2.4.1
